module/Application/src/Module.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @link      http://github.com/zendframework/ZendSkeletonApplication for the canonical source repository
+ * @copyright Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Application;
+
+use Zend\Mvc\ModuleRouteListener;
+use Zend\Mvc\MvcEvent;
+
+class Module
+{
+    const VERSION = '3.0.0dev';
+
+    public function onBootstrap(MvcEvent $e)
+    {
+        $moduleRouteListener = new ModuleRouteListener();
```

Comment:
Interestingly, more than one pro services consultant has recently asked about `ModuleRouteListener` with recent zend-mvc releases, as clients were using it. I'll remove it, but I'm going to make a migration note that it's removed for new skeletons, and that users should be using fully qualified class names when referring to controllers and middleware.